srcpkgs/xmonad-contrib/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'xmonad-contrib'
+pkgname=xmonad-contrib
+version=0.13
+revision=1
+build_style=haskell-stack
+stackage=lts-11.17
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXinerama-devel libXrandr-devel libXft-devel"
+short_desc="Third party extensions for xmonad"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="BSD-3-Clause"
+homepage="http://xmonad.org/"
+distfiles="http://hackage.haskell.org/package/${pkgname}-${version}/${pkgname}-${version}.tar.gz"
```

Comment:
it also might be https but i didn't test like homepage=

Reply:
It is https if you connect to the homepage on hackage in https; and http otherwise, I just copied the link...